apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -486,7 +486,7 @@ type FairSharing struct {
 // +genclient:nonNamespaced
 // +kubebuilder:object:root=true
 // +kubebuilder:storageversion
-// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:resource:scope=Cluster,shortName={cq}
```

Comment:
Actually, we dropped this shortName here: https://github.com/kubernetes-sigs/kueue/pull/604

So, I'm curious about whether we should re-enable this short name.
@alculquicondor  WDYT?

Reply:
Do you remember the reason for that? 

In theory `kueuectl` can hack the args that are actually passed to `kubectl` in pass through commands , the problem is that `get cq` will end up working with `kueuectl` and create confusion when it comes to UX.

/cc @mwielgus

Reply:
Actually, I don't remember the reason...

Reply:
I just considered them too small and to potentially run into conflicts with CRDs from different projects.

But maybe it's worth, given a trend for users to use kueue without being too familiar with kubernetes, so I wouldn't expect many conflicts.